modules/installation-vsphere-infrastructure.adoc

```diff
@@ -6,21 +6,25 @@
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6.5
-or 6.7U2 or later instance.
+You must install the {product-title} cluster on a VMware vSphere version that meets the requirements for the components that you use.
 
-VMware recommends using vSphere Version 6.7 U2 or later with your {product-title}
-cluster.
-vSphere 6.7U2 includes:
+.Minimum supported vSphere version for VMware components
+|===
+|Component | Minimum supported versions |Description
 
-* Support for VMware NSX-T
-* Support for vSAN, VMFS and NFS, using the in-tree VCP
+|Hypervisor
+|vSphere 6.5 with HW version 13
+|This version is the is the minimum version that {op-system-first} supports. See the link:https://access.redhat.com/ecosystem/search/#/ecosystem/Red%20Hat%20Enterprise%20Linux?sort=sortTitle%20asc&vendors=VMware&category=Server[Red Hat Enterprise Linux 8 supported hypervisors list].
 
-While vSphere 6.5 with Hardware version 13 is supported, {product-title}
-clusters are subject to the following restrictions:
+|Networking (NSX-T)
+|vSphere 6.5U3 or vSphere 6.7U2+
+|Previous versions of vSphere are not supported.
 
-* NSX-T SDN is not supported.
-* You must use another SDN or storage provider that {product-title} supports.
+|Storage (vSphere Container Storage Interface plug-in)
+|vSphere 6.7U3
```

Comment:
@duanwei33 
Do u happen to know if this storage pug-in is a must for ocp 4.z? I'm not quite sure about this component requirement.

Reply:
@jiajliu 
I understand it specifies the minimum supported vSphere versions to install the vSphere CSI driver, and refer to the description for CSI doc, looks like the 6.7U3 is the right version(As of the time of this writing :)). 
https://github.com/kubernetes-sigs/vsphere-csi-driver	

> @duanwei33
> Do u happen to know if this storage pug-in is a must for ocp 4.z? I'm not quite sure about this component requirement.

@jiajliu 
I understand it specifies the minimum supported vSphere versions to install the vSphere CSI driver, and refer to the description for CSI doc, looks like the 6.7U3 is the right version(As of the time of this writing :)). 
https://github.com/kubernetes-sigs/vsphere-csi-driver

Reply:
Thx @duanwei33 's confirm. Then this CSI part is lgtm.

Reply:
I thought the Cloud provider integration was using intree storage connectivity for vsphere storage and not CSU driver's. Is this changing with 4.4? Who is providing the driver's, are we pulling these from Vsphere? How will this work for disconnected installs?

Reply:
@duanwei33, do you have answers to Eric's questions?

> I thought the Cloud provider integration was using intree storage connectivity for vsphere storage and not CSU driver's. Is this changing with 4.4? Who is providing the driver's, are we pulling these from Vsphere? How will this work for disconnected installs?

Reply:
@sferich888 @kalexand-rh No change, still use in-tree storage for vSphere, I think it just indicates the minimum vSphere infrastructure requirements version if customers want to install CSI driver on vSphere. It says "This plug-in creates vSphere storage by using the standard Kubernetes CSI interface. Previous versions of vSphere do not provide the required API support for the plug-in." 
I am not sure when we start to list the CSI part here, or maybe add something like "not covered by OCP" as Eric's concern, WDYT?
BTW, for the version support, from my view, it's better to give a link to VMware doc for an official one.

Reply:
I just see @listey had the related comments for CSI part.
https://github.com/openshift/openshift-docs/pull/19866#issuecomment-592036239

Reply:
How would one go about switching from the in-tree driver to the CSI driver?

Reply:
AFAIK, the VMware guys are working on an operator for it. The process is pretty similar to the cloud provider.

Reply:
Is it something we should be documenting how to use here? It crosses a gray line with partners @aheslin